README.md

```diff
@@ -1,39 +1,35 @@
 # k6-jslib-utils
+
 A collection of small utility functions useful during load testing with k6.
 
 Docs: http://k6.io/docs/javascript-api/jslib/utils
 
 Download the latest release from https://jslib.k6.io/
 
-
 ## Example
 
 ```javascript
-import { sleep } from 'k6';
-import http from 'k6/http';
+import { sleep } from 'k6'
```

Comment:
This makes me think we are not using the same configuration as we have for https://github.com/grafana/k6-docs examples. I guess they should be consistent.

Reply:
k6 docs use a mix of ESLint and Prettier as far as I could see. Also, as we don't have a unified config, should we consider k6-docs as the de-facto standard?